Example/openxcframework/StatusViewController.swift

```diff
@@ -10,28 +10,28 @@ import openXCiOSFramework
 class StatusViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
 
     // UI Labels
-    @IBOutlet weak var actConLab: UILabel!
-    @IBOutlet weak var msgRvcdLab: UILabel!
-    @IBOutlet weak var verLab: UILabel!
-    @IBOutlet weak var devidLab: UILabel!
-    @IBOutlet weak var platformLab: UILabel!
-    @IBOutlet weak var throughputLab: UILabel!
-    @IBOutlet weak var averageMessageLab: UILabel!
-    @IBOutlet weak var NetworkImg: UIImageView!
+    @IBOutlet weak var activeConnectionLabel: UILabel!
+    @IBOutlet weak var messagesReceivedLabel: UILabel!
+    @IBOutlet weak var versionLabel: UILabel!
+    @IBOutlet weak var deviceIdLabel: UILabel!
+    @IBOutlet weak var platformLabel: UILabel!
+    @IBOutlet weak var throughPutLabel: UILabel!
+    @IBOutlet weak var averageMessageLabel: UILabel!
+    @IBOutlet weak var networkImageView: UIImageView!
     
-    fileprivate var troughputLoop: Timer!
+    fileprivate var throughPutLoop: Timer!
     // scan/connect button
-    @IBOutlet weak var searchBtn: UIButton!
+    @IBOutlet weak var searchButton: UIButton!
     // disconnect button
-    @IBOutlet weak var disconnectBtn: UIButton!
+    @IBOutlet weak var disConnectButton: UIButton!
```

Comment:
As above, the variable name should be disconnectButton

Reply:
Done